src/components/AppLayout/Notifications/Notification.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { TNotificationObject } from "./types";
+
+const Notification = ({
+    icon,
+    title,
+    message,
+    buttonAction,
```

Comment:
```suggestion
    actionButton,
```

Just wanted to make it same as the actionText

Reply:
this meant to be the action of button which can we buttonAction
actionButton means a different thing